### PR TITLE
create the team page.

### LINKED ISSUE #2401: Create /team page
URL suggestions: 
- about/team

Notes:
- Sync with Isin for the map integration as it builds upon her work
- ping @chrifro before merging the PR who will take care that we have everyone's consent before publishing personal pictures
- add the new page to the footer under `company`

View [mockup](https://www.figma.com/file/K0VV8lc3pmlMVModYM70nj/Website?node-id=5550%3A8290)

### PR DESCRIPTION
**Pull request by nisarhassan12**
Fixes #2401 

- Can be tested on [`/about/team`](https://deploy-preview-2446--gitpod-kumquat.netlify.app/about/team)

### Todos

- Add the scrollbar to life at Gitpod. 

<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/2446"></a>

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/2446"></a>

### CODE REVIEW

**Comment by nisarhassan12** [comment]
The map does not look as exactly in the mockup i.e because it's an embed coming from Google data studio i.e we can't make it compatible with the dark mode or remove the widgets that are part of the embed. 
image

**Review by Cahllagerfeld — COMMENTED** [review]
Are the numbers of Gitpodders displayed on the map up to date, and who is responsible to update them? 😅

**Comment by nisarhassan12** [comment]
> Are the numbers of Gitpodders displayed on the map up to date, and who is responsible to update them? 😅

I got the map from Isin Pesch so I guess she is the one responsible for updating them.

File: src/routes/about/team.svelte [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    @apply w-full max-w-5xl h-[300px] sm:h-[400px] md:h-[580px] xl:h-[630px] 2xl:h-[660px];
+  }
+</style>
+
+<OpenGraph
+  data={{
+    title: "",
+    description: "",
+    keywords: "",
+  }}
```
Comment (nisarhassan12) [inline src/routes/about/team.svelte:30]: @MichaelAring Can you suggest some values for these?

File: src/routes/about/team.svelte [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+>
+  <iframe
+    title="World Map of where different people working at Gitpod are located."
+    slot="content"
+    src="https://datastudio.google.com/embed/reporting/4082b7be-0033-437f-9925-4ed2fb6f80a0/page/NaupC"
+    frameborder="0"
+    style="border:0"
+    allowfullscreen
+    class="mx-auto mt-x-small md:mt-small"
+  />
```
Comment (Siddhant-K-code) [inline src/routes/about/team.svelte:46]: Can we fix its dimension? So that, it doesn't show scroll at right (in frame)

**Comment by nisarhassan12** [comment]
@jacobparis Do you know what direction we could take to fix the map here embed here? Thanks

**Comment by jacobparis** [comment]
@nisarhassan12 Rather than settings heights based on the breakpoints, I set the aspect ratio so the height is automatically calculated. Experimented a bit and `64/50` seems to be the shortest height that does not trigger scrollbars. Tested on Chrome and Safari with a variety of screen widths and zoom levels

image
image

File: src/routes/about/team.svelte (hunk 1 above)
Comment (jacobparis) [reply src/routes/about/team.svelte:24]: I filled it in

**Comment by jacobparis** [comment]
I also added the open graph meta copy

**Comment by nisarhassan12** [comment]
Thanks, @jacobparis I just tested the deploy preview while it works for Chrome/Safari I still see the scrollbars across various screen sizes see the below:

https://user-images.githubusercontent.com/46004116/189643863-33e74585-d711-4eda-961f-dda84d975698.mov

**Comment by jacobparis** [comment]
I don't get that at any screen size, can you try incrementing the aspect ratio until they go away? Not sure exactly what the difference would be between my screen and yours
image

**Comment by nisarhassan12** [comment]
> can you try incrementing the aspect ratio until they go away?
@jacobparis They go away when it is `72/72` or `50/50` on larger screen sizes and as I go to smaller screen sizes they come back again.

**Comment by chrifro** [comment]
Review cc @nisarhassan12 :

- [ ] the world map takes quite long to load. Is there anything we can do to reduce the time?
-  [ ] enable vertical scrolling for the pictures under "life at Gitpod".
-  [ ] the loading for new pictures under "life at Gitpod" takes quite long and the layout changes. 
-  [ ] mobile design: remove  image so that the text has more space
Bildschirmfoto 2022-09-15 um 11 30 15

-  [ ] mobile: adjust images for mobile (radius: 8px, spacing 8px)
Bildschirmfoto 2022-09-15 um 11 30 00

idea regarding the world map: could we export an updated png of the world map and implement that instead of the embed? we don't need the interaction on hover if that's too complicated. We could set a reminder to update the image every 6 months. The purpose of the image is to underline that we are working remotely from everywhere in the world. Accurate data at all time is not needed for that.

When do you think we can get this merged @nisarhassan12 (it's open since a long time)

**Comment by nisarhassan12** [comment]
>  - [ ] the world map takes quite long to load. Is there anything we can do to reduce the time?

@chrifro unfortunately we can't do anything to make it load faster i.e we don't control the code that gets shipped with the embed. The only thing that we could do is that we implement it ourselves or else as you suggested use a png image.

**Comment by chrifro** [comment]
>unfortunately we can't do anything to make it load faster i.e we don't control the code that gets shipped with the embed. The only thing that we could do is that we implement it ourselves or else as you suggested use a png image.

Let's go for the png version then as a skateboard 🛹

**Comment by chrifro** [comment]
>Not sure whether we should wait for Isin to get back from vacation for a better quality of map screenshots.

- [ ] Could you decrease the image size to be closer to [the mockup](https://www.figma.com/file/K0VV8lc3pmlMVModYM70nj/Website?node-id=5550%3A8290)? 🙏

- [ ] update the cropped image as discussed in Figma
- [ ] update the image order as marked in Figma

Afterwards I'd share the preview with the team to get the personal images approved. The world map image can be swapped out once Isin is back next week.

**Comment by nisarhassan12** [comment]
@chrifro I have made the requested changes. Please have a look when you get time. Thanks

**Comment by chrifro** [comment]
@nisarhassan12 Can you get in contact with Isin to resolve the world map issue once she is back from vacation? 
 
 I'll take care of the approval of the images

**Comment by nisarhassan12** [comment]
Yeah sure. I can do that.

**Comment by chrifro** [comment]
Could you delete this section as suggested by Philipp? [[1](https://gitpod.slack.com/archives/C019K7PND1A/p1663340700816969?thread_ts=1663331894.344829&cid=C019K7PND1A)] 
Bildschirmfoto 2022-09-21 um 10 41 09

**Comment by chrifro** [comment]
Outstanding to-do's:

- [ ] remove this section [[1](https://github.com/gitpod-io/website/pull/2446#issuecomment-1253390508)]
- [ ] update world map with SVG image (see [Figma](https://www.figma.com/file/K0VV8lc3pmlMVModYM70nj/Website?node-id=5550%3A8290))

Very sorry about the multiple world map versions. Hope this one will work fine now. Thanks for your patience 🙏

Once that has been addressed we can merge the page. 

@nisarhassan12 do you still have time to take care of this? 🙏

**Pull request merged**